CONTRIBUTING.md

```diff
@@ -64,16 +62,14 @@ a good reason is "This violates the style guide, but it improves type safety."
 Strive for commits that implement a single or related set of functionality, using as many commits as is necessary (more is better).
 That said, please ensure that the repository compiles and passes tests without error for each commit in your pull request.  Note
 that to be accepted into the repository, the pull request must [pass all tests](#testing your changes) on all supported platforms
--- the automatic Github Travis and AppVeyor continuous integration features will assist in enforcing this requirement.
 
 #### Generated Source Code
 
 The `icd/generated` directory contains source code that is created by several
 generator scripts in the `scripts` directory. All changes to these scripts _must_ be submitted with the
 corresponding generated output to keep the repository self-consistent. This requirement is enforced by both
```

Comment:
```suggestion
corresponding generated output to keep the repository self-consistent. This requirement is enforced by ~both~ CI.
```
Both is redundant, since there is only github actions doing the check.